internal/website/gatherexamples/gatherexamples.go

```diff
@@ -141,11 +152,16 @@ func gather(pkgs []*packages.Package) map[string]example {
 					Comments: file.Comments,
 				})
 				if err != nil {
-					panic(err) // will only occur for bad invocations of Fprint
+					return nil, err // will only occur for bad invocations of Fprint
 				}
 				original := sb.String()
-				if !strings.Contains(original, "\n\t"+signifierCommentPrefix) {
-					// Does not contain the signifier comment. Skip it.
+				if !strings.Contains(original, signifierComment) {
+					// Does not contain the signifier comment. Skip it, but first verify
+					// that it doesn't contain any PRAGMA comments; only examples with
+					// the signifier should include pragmas.
+					if strings.Contains(original, pragmaPrefix) {
+						return nil, fmt.Errorf("%s in package %s has PRAGMA(s) for gatherexamples, but is not marked for inclusion with \"Package this example for gocloud.dev.\"", fn.Name.Name, pkg.PkgPath)
+					}
```

Comment:
That's a helpful error.

Reply:
Thanks! I found a few examples with it that had these (otherwise strange-looking) comments for no reason.